ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -465,6 +465,36 @@ FCU_DFSS_RTC_RHC_RHDHC_DXRC:
   - RHDHC
   - DXRC
 
+FCU_DFSS_RTC_RHC_RHDHC_DXRC_HTRC_FDPM:
```

Comment:
is this type actually being used? would expect the two CAC A/B units to match to the type below this

Reply:
CAC B does not have the zone monitoring points

Reply:
Let's confirm on site that A and B serve the same zone - if so, need to duplicate the zone points for both entities and then remove this canonical type